tfimm/train/datasets.py

```diff
@@ -3,7 +3,7 @@
 import tensorflow as tf
 import tensorflow_datasets as tfds
 
-from tfimm.train.registry import cfg_serializable
+from .registry import cfg_serializable
```

Comment:
By making the imports relative, we make it possible to copy the files from `tfimm` to another repository and use them without change. In case we want to use the framework, but need to make changes to it quickly. Or just want to work on it locally in the context of another project and them upstream the changes back to `tfimm` later.

Reply:
Fair point :)